src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -68,25 +74,97 @@
+                               (first (rf/sub [:contacts/contact-two-names-by-identity chat-id]))
+                               (str emoji " " chat-name))
+        {:keys [bio]}        (rf/sub [:contacts/contact-by-identity chat-id])
+        loading-messages?    (rf/sub [:chats/loading-messages? chat-id])
+        all-loaded?          (rf/sub [:chats/all-loaded? chat-id])
+        online?              (rf/sub [:visibility-status-updates/online? chat-id])
+        contact              (when-not group-chat
+                               (rf/sub [:contacts/contact-by-address chat-id]))
+        photo-path           (when-not (empty? (:images contact))
+                               (rf/sub [:chats/photo-path chat-id]))]
+    [:f>
+     (fn []
+       (let [border-animation            (reanimated/interpolate scroll-y
+                                                                 [30 125]
+                                                                 [14 0]
+                                                                 header-extrapolation-option)
+             image-scale-animation       (reanimated/interpolate scroll-y
+                                                                 [50 125]
+                                                                 [1 0.5]
+                                                                 header-extrapolation-option)
+             image-top-margin-animation  (reanimated/interpolate scroll-y
+                                                                 [50 125]
+                                                                 [0 40]
+                                                                 header-extrapolation-option)
+             image-side-margin-animation (reanimated/interpolate scroll-y
+                                                                 [50 125]
+                                                                 [0 -20]
+                                                                 header-extrapolation-option)]
+         [:<>
+          [rn/view {:style (style/header-container insets)}
+           (when cover-bg-color
+             [rn/view {:style (style/header-cover cover-bg-color status-bar-height)}])
+           [reanimated/view {:style (style/header-bottom-part border-animation)}
+            [rn/view {:style style/header-avatar}
+             (when-not group-chat
+               [rn/view {:style {:align-items :flex-start}}
+                [reanimated/view
+                 {:style (style/header-image image-scale-animation
+                                             image-top-margin-animation
+                                             image-side-margin-animation)}
+                 [quo/user-avatar
+                  {:full-name       display-name
+                   :online?         online?
+                   :profile-picture photo-path
+                   :size            :big}]]])
```

Comment:
What if `group-chat`?

Reply:
The avatar doesn't get displayed in this case. Just a name of the chat.